.circleci/config.yml

```diff
@@ -6,5 +6,25 @@ jobs:
     steps:
       - checkout
       - run:
-          name: Run tests
+          name: Prepare Python env # install python3.7, python3-pip and pipenv for pyspark.
+          command: |
+            sudo apt-get update
+            sudo apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+            sudo apt install libedit-dev
+            sudo apt install python3-pip --fix-missing
+            sudo pip3 install pipenv
+            curl https://pyenv.run | bash
+            export PATH="/home/circleci/.pyenv/bin:$PATH"
+            eval "$(pyenv init -)"
+            eval "$(pyenv virtualenv-init -)"
+            pyenv install 3.7.4
+            pyenv global system 3.7.4
+            pipenv install
+      - run:
+          name: Run Pyhton tests # build the Delta Lake and run the python tests
+          command: |
+            build/sbt "++2.11.12 package"
+            pipenv run spark-submit --jars target/scala-2.11/delta-core_2.11-0.3.1-SNAPSHOT.jar src/test/python/test_deltatable.py
```

Comment:
Does `target/scala-2.11/delta-core_2.11-*-SNAPSHOT.jar` work?

Reply:
yes, it works, the failure before was caused by wrong order of option params.

Reply:
changed.